### PR TITLE
Adds a changelog for "strip trailing newline" bug fix

### PR DESCRIPTION
**Pull request by oguzkocer**
I was planning to do a minor release and realized there wasn't a changelog for https://github.com/wordpress-mobile/release-toolkit/pull/297, so I added the title of that PR to the Bug fixes section. I am no longer planning to do the release, but I think it's still good to have that in the changelog.

P.S: It turns out this is my first PR in this project 😱

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ _None_
 
 ### Bug Fixes
 
+* Strip trailing new lines in single line msgid when generating .po[t] file
```
Comment (AliSoftware) [inline CHANGELOG.md]: nit: when we have it I think it's nice to to provide a ref to the related PR to link back to any interesting discussion about the change

```suggestion
* Strip trailing new lines in single line msgid when generating .po[t] file. [#297]
```

**Review by AliSoftware — APPROVED** [review]
Left a suggestion; once fixed should be good to land 👍

**Comment by oguzkocer** [comment]
I assume someone started to add Buildkite to this repo, but I don't see a .buildkite folder yet, so I am going to proceed with merging this.

**Comment by AliSoftware** [comment]
Yup, @jkmassel have [a PR opened to add BuildKite](https://github.com/wordpress-mobile/release-toolkit/pull/299) but not merged yet hence why no `.buildkite` folder in `develop` 👍

**Comment by oguzkocer** [comment]
Oh, I see how it is. He realized I have no contribution to this project, so didn't add me as a reviewer 😂

**Pull request merged**